Cargo.toml

```diff
@@ -8,13 +8,19 @@ license = "MIT OR Apache-2.0"
 members = ["", "fuzz"]
 
 [workspace.dependencies]
-arbitrary = { version = "1.2.3", features = ["derive"] }
-bolero = "0.8.0"
-borsh = { version = "0.10.2", features = ["rc"] }
-libfuzzer-sys = { version = "0.4", features = ["arbitrary-derive"] }
+arbitrary = { version = "1", features = ["derive"] }
+bolero = ">=0.8.0, <0.10.0"
+borsh = { version = "0.10", features = ["rc"] }
+libfuzzer-sys = { version = ">=0.2.0, <0.5", features = ["arbitrary-derive"] }
 near-account-id = { path = "", features = ["internal_unstable"] }
-serde = { version = "1.0.136", features = ["alloc", "derive", "rc"] }
-serde_json = "1.0.68"
+serde = { version = "1.0.119", features = ["alloc", "derive", "rc"] }
+serde_json = "1.0.25"
+
+# We don't use these directly - setting these version requirements is a workaround to block dependency resolution
+# from choosing dependency versions that will cause compilation errors.
+anyhow = "1.0.14"
+backtrace = "0.3.13"
+lazy_static = "1.0.2"
```

Comment:
Do older versions of the packages case compilation errors or what? If so, please, be more specific that it is caught in minimal-deps CI.

Will any of these be compiled when no default features are enabled?

Reply:
Yes, older versions of those deps cause compilation errors. It was caught by the minimal-deps CI.

> Will any of these be compiled when no default features are enabled?
I forgot about that. I'll check.

Reply:
They're always present in test builds. We're fine.